Formula/podman-compose.rb

```diff
@@ -0,0 +1,49 @@
+class PodmanCompose < Formula
+  include Language::Python::Virtualenv
+
+  desc "Alternative to docker-compose using podman"
+  homepage "https://github.com/containers/podman-compose"
+  url "https://github.com/containers/podman-compose/archive/refs/tags/v1.0.3.tar.gz"
```

Comment:
overall, LGTM, just one QQ, can it use `https://files.pythonhosted.org/packages/c7/aa/0997e5e387822e80fb19627b2d4378db065a603c4d339ae28440a8104846/podman-compose-1.0.3.tar.gz`?